### PR TITLE
EEG module has been added

### PR DESCRIPTION
**Pull request by kcho**

### CODE REVIEW

**Comment by tashrifbillah** [comment]
Hi Kevin, I see that #15 also attempts to merge against `master`. Do you know if these two are independent? Otherwise, merging one first may result in conflict.

File: dpanonymize/__init__.py [hunk 1]
```diff
@@ -56,7 +58,7 @@ def anonymize(self, **kwargs) -> None:
             module.remove_pii(self.file_path, self.general_path)
 
     def __repr__(self):
-        return f"<{self.file_path.name}>"
+        return f"<{self.file_path.name} {self.dtype}"
```
Comment (tashrifbillah) [inline dpanonymize/__init__.py:61]: Why is the `self.dtype` necessary?

File: dpanonymize/eeg/__init__.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from pathlib import Path
+import shutil
+from typing import Union
+
+def remove_pii(data_in: Union[str, Path], data_out: Union[str, Path]):
```
Comment (tashrifbillah) [inline dpanonymize/eeg/__init__.py:5]: This is just a placeholder, so we won't have to review much ...

**Review by tashrifbillah — APPROVED** [review]
Asked one question ...

File: dpanonymize/__init__.py (hunk 1 above)
Comment (kcho) [reply dpanonymize/__init__.py:61]: Just to print and see if the functions are working well detecting the correct data type for each file path. Mostly for myself.

**Comment by kcho** [comment]
> Hi Kevin, I see that #15 also attempts to merge against `master`. Do you know if these two are independent? Otherwise, merging one first may result in conflict.

Hi Tashrif,

Yes, I wanted to separate these two, #14 for EEG module and #15 for dtype detection, but I may have made some mistakes in committing changes in both branches. We should try merging #14, then #15.

**Pull request merged**